pkg/plugin/twinmaker/client.go

```diff
@@ -88,9 +84,8 @@ func NewTwinMakerClient(settings models.TwinMakerDataSourceSetting) (TwinMakerCl
 	stsSettings.AssumeRoleARN = ""
 
 	stsSessionConfig := awsds.SessionConfig{
-		Settings:      stsSettings,
-		HTTPClient:    httpClient,
-		UserAgentName: &agent,
```

Comment:
Does the STS client not need the agent name set?

Reply:
We really only care about the TwinMaker client, since we're the service that will be tracking Grafana usage. We never needed to set the UserAgent header of any other AWS client.

Reply:
I'd lean towards continuing to set the agent just to reduce the change, but I don't think it matters that much.